Common/src/containers/CFileReaderLUT.cpp

```diff
@@ -238,15 +240,67 @@ void CFileReaderLUT::ReadRawDRG(const string& file_name) {
 
 }
 
-string CFileReaderLUT::SkipToFlag(ifstream* file_stream, const string& flag) {
-  string line;
-  getline(*file_stream, line);
+/**
+ * @brief Searches for the position of flag in file_stream and 
+ *        sets the stream position of file_stream to that position.
+ * 
+ * @param file_stream 
+ * @param current_line 
+ * @param flag 
+ */
+void CFileReaderLUT::SkipToFlag(ifstream* file_stream, const string& current_line, const string& flag) {
```

Comment:
These helper methods should be either const, or static, or free functions in an anonymous namespace.
For performance they should be inline.
Please pass file_stream by reference

Reply:
OK, I think making them const makes the most sense, it is also what we do with the other functions. Are you asking to make them inline? They only go through the relatively small header file, so there are no performance issues.

Reply:
@pcarruscag Thank you for your comments, Pedro. I think functions defined in classes are implicitly inline anyways, aren't they? Happy to add some inline specifiers to be sure, though!

Reply:
They are implicitly inline if you define them in the body of the class, if you declare them in the body of the class and define them in a cpp they are not inline.
However, maybe the compiler can still inline within the cpp where they are defined (they will def. not be inlined in other places that only see the declaration of the function but not its definition, e.g. other cpps, unless you turn on link-time optimizations).
But like Nijso said, probably not important here.

Reply:
Cool, thank you!